src/System.Windows.Forms/src/System/Windows/Forms/FontDialog.cs

```diff
@@ -155,11 +153,7 @@ public Font Font
         {
             get
             {
-                Font result = font;
-                if (result is null)
-                {
-                    result = Control.DefaultFont;
-                }
+                Font? result = font ?? Control.DefaultFont;
```

Comment:
`Control.DefaultFont` can't be null
https://github.com/dotnet/winforms/blob/09120ef4871dccfa3b4c09e33ad96648cf3f4c64/src/System.Windows.Forms/src/System/Windows/Forms/Control.cs#L1811-L1823
```suggestion
                Font result = font ?? Control.DefaultFont;
```

Reply:
`SystemFonts.MessageBoxFont` is nullable though. I guess when we enable nullability in Control we could use the null-forgiving operator there.

Reply:
Ah, right. All good then